### ISSUE TITLE
Platform name is coming as "unknown" while doing mbedls in UBUNTU 16.04LTS

### ISSUE THREAD

**Issue by Vanireddy6** [ARMmbed/mbed-os-tools#121]
we are using our own company platforms.
while doing mbedls, platform name is coming as "unknown".
```
cy-user@cyuser-Latitude-E6430:~$ mbedls
WARNING:mbedls.lstools_base:daplink entry: "1901" not found in platform database
| platform_name | platform_name_unique | mount_point            | serial_port  | target_id                                        | daplink_version |
|---------------|----------------------|------------------------|--------------|--------------------------------------------------|-----------------|
| unknown       | None[0]              | /media/cy-user/DAPLINK | /dev/ttyACM0 | 190100006c0f181f036c0f1800000000000000002e127069 | 0251            |
```

**Comment by ciarmcom** [comment]
*Internal Jira reference: https://jira.arm.com/browse/MBOCUSTRIA-820*

**Comment by bridadan** [comment]
Hi @Vanireddy6, looks like it isn't finding the entry for `1901`. From your log:

```
WARNING:mbedls.lstools_base:daplink entry: "1901" not found in platform database
```

This target support was released with mbed-ls version `1.7.1`, can you please be sure you're at the latest release? `pip install -U mbed-ls`.
